config-migrate.go

```diff
@@ -65,23 +68,25 @@ func migrateV2ToV3() {
 	if cv2.Version != "2" {
 		return
 	}
-	serverConfig.SetAddr(":9000")
-	serverConfig.SetCredential(credential{
+	srvConfig := &configV3{}
+	srvConfig.Version = "3"
+	srvConfig.Addr = ":9000"
+	srvConfig.Credential = credential{
 		AccessKeyID:     cv2.Credentials.AccessKeyID,
```

Comment:
Use localized server config instead of global config. We load freshly at a later stage.